micrometer-jakarta/build.gradle

```diff
@@ -0,0 +1,116 @@
+plugins {
+    id 'me.champeau.mrjar' version "0.1.1"
+}
+
+description 'Module for Jakarta based instrumentations'
+
+jar {
+    bundle {
+
+        bnd '''\
+        Import-Package: \
+            org.eclipse.jetty.*;resolution:=dynamic;version="${@}",\
+            org.glassfish.jersey.*;resolution:=dynamic;version="${@}",\
+            ch.qos.logback.*;resolution:=dynamic;version="${@}",\
+            org.apache.logging.log4j.*;resolution:=dynamic;version="${@}",\
+            jakarta.jms.*;resolution:=dynamic;version="${@}",\
+            jakarta.servlet.*;resolution:=dynamic;version="${@}",\
+            io.micrometer.context.*;resolution:=dynamic,\
+            io.micrometer.observation.*;resolution:=dynamic;version="${@}",\
+            *
+        '''.stripIndent()
+    }
+}
+
+dependencies {
+    api project(":micrometer-core")
+    api project(":micrometer-commons")
+    api project(":micrometer-observation")
+    optionalApi 'jakarta.ws.rs:jakarta.ws.rs-api' // jakarta
+
+    // Aspects
+    optionalApi 'org.aspectj:aspectjweaver'
+
+    optionalApi 'org.eclipse.jetty:jetty-server'
+    // jakarta servlet
+    optionalApi 'jakarta.servlet:jakarta.servlet-api'
+    optionalApi 'org.eclipse.jetty:jetty-client:12.0.+' // can't move to dependencies.gradle, we have 2 versions of this lib (check micrometer-core)
+    optionalApi 'org.apache.tomcat.embed:tomcat-embed-core:10.1.+' // can't move to dependencies.gradle, we have 2 versions of this lib (check micrometer-core)
+    optionalApi 'org.glassfish.jersey.core:jersey-server:3.1.+' // can't move to dependencies.gradle, we have 2 versions of this lib (check micrometer-core)
+    // jakarta JMS
+    optionalApi 'jakarta.jms:jakarta.jms-api'
+
+    // log monitoring
+    optionalApi 'ch.qos.logback:logback-classic'
+    optionalApi 'org.apache.logging.log4j:log4j-core'
+
+    optionalApi project(':micrometer-observation')
+    optionalApi 'io.micrometer:context-propagation'
```

Comment:
I don't see any context-propagation specific instrumentation. This should be removed. `micrometer-observation` is already declared as `api` so it can be removed from `optionalApi`.